package.json

```
@@ -0,0 +1,94 @@
+{
+  "name": "manh-react-survey",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^1.4.0",
+    "i18next": "^22.5.1",
+    "i18next-browser-languagedetector": "^7.0.2",
+    "i18next-http-backend": "^2.2.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-i18next": "^12.3.1",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "5.0.1",
+    "sass": "^1.49.11"
+  },
+  "scripts": {
+    "start": "react-scripts -r @cypress/instrument-cra start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --transformIgnorePatterns",
+    "eject": "react-scripts eject",
+    "test:coverage": "rimraf coverage && react-scripts test --coverage --watchAll=false && npm run cypress:run && node ./scripts/coverage-merge.js && nyc report",
+    "test:merge-coverage": "node ./scripts/coverage-merge.js",
+    "lint": "eslint ./ --ext .js,.ts,.tsx",
+    "lint:fix": "eslint ./ --ext .js,.ts,.tsx --fix",
+    "stylelint": "stylelint '**/*.scss'",
+    "stylelint:fix": "stylelint '**/*.scss' --fix",
+    "codebase:lint": "npm run lint && npm run stylelint",
+    "codebase:fix": "npm run lint:fix && npm run stylelint:fix",
+    "cypress": "start-server-and-test start 3000 cypress:run",
+    "cypress:run": "cypress run",
+    "cypress:open": "cypress open"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.0-placeholder-for-preset-env.2",
+    "@cypress/browserify-preprocessor": "3.0.2",
+    "@cypress/code-coverage": "^3.9.12",
```

could we use the exact version? The idea is: using an exact version prevents unwanted minor/patch updates. 

```suggestion
    "@cypress/code-coverage": "3.9.12",
```

---

Updated in [e5c5101](https://github.com/manh-t/react-survey/pull/39/commits/e5c5101285ed5f2dd852132b3fd130bd9dddf7dc)